make/modules/java.base/gensrc/GensrcVarHandles.gmk

```diff
@@ -168,12 +168,27 @@ endef
 # Setup a rule for generating a memory segment var handle view class
 # Param 1 - Variable declaration prefix
 # Param 2 - Type with first letter capitalized
+# TODO decide if CAS/Bitwise/AtomicAdd should be enabled for subints
```

Comment:
Exactly - let's keep this out for now

Reply:
I have disabled them with #, and the status is confirmed by test for access modes. I kept the infra to make future reenabling easy.

Reply:
> I have disabled them with #, and the status is confirmed by test for access modes. I kept the infra to make future reenabling easy.

Doh - I missed the `#` -- maybe add few more to make that more explicit? (I agree with the approach)

Reply:
Left a space and an extra note to make the comment hash more obvious.

Reply:
Unless you plan to shortly push a new PR where you either enable this functionality, or remove the commented-out lines, I strongly prefer *not* to have commented-out code in the makefiles. It's not that hard to re-create these lines should they be needed. And you can always find the history in this PR.